apps/spruce/src/pages/UserPatches.tsx

```diff
@@ -36,7 +29,8 @@ export const UserPatches = () => {
       userId,
       patchesInput: {
         ...patchesInput,
-        includeCommitQueue: isCommitQueueCheckboxChecked,
```

Comment:
This parameter is no longer used from Spruce. Should it be deprecated from the GQL API?

Reply:
I'll file a ticket for this